xsgen/openmc_origen.py

```diff
@@ -151,15 +153,16 @@ def __init__(self, rc):
         self.eafds = data_source.EAFDataSource()
         self.omcds = data_source.OpenMCDataSource(
                         cross_sections=rc.openmc_cross_sections,
-                        src_group_struct=np.logspace(1, -9, 1001))
+                        src_group_struct=rc.openmc_group_struct)
         data_sources = [self.omcds]
         if not rc.is_thermal:
             data_sources.append(self.eafds)
-        data_sources += [data_source.SimpleDataSource(),
-                         data_source.NullDataSource()]
-        for ds in data_sources[1:]:
+        #data_sources += [data_source.SimpleDataSource(),
+        #                 data_source.NullDataSource()]
+        for ds in data_sources[0:]:
```

Comment:
`data_sources[0:]` is the same as `data_sources[:]`, which in a loop is equivalent to `data_sources`